pkg/apis/core/validation/seed.go

```diff
@@ -114,6 +114,10 @@ func ValidateSeedSpec(seedSpec *core.SeedSpec, fldPath *field.Path, inTemplate b
 	allErrs = append(allErrs, cidrvalidation.ValidateCIDRParse(networks...)...)
 	allErrs = append(allErrs, cidrvalidation.ValidateCIDROverlap(networks, networks, false)...)
 
+	vpnDefaultRanges := []cidrvalidation.CIDR{cidrvalidation.NewCIDR(v1beta1constants.DefaultVpnRange, field.NewPath("vpn-default"))}
```

Comment:
```suggestion
	vpnDefaultRanges := []cidrvalidation.CIDR{cidrvalidation.NewCIDR(v1beta1constants.DefaultVpnRange, field.NewPath(""))}
```

There is no `vpn-default` field in the API, so better leave it empty to prevent confusion.

Reply:
Done.